test-resources/xmir-parsing-packs/simple.yaml

```
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+xmir: |
+  <object
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/objectionary/eo/refs/heads/gh-pages/XMIR.xsd"
+  >
+    <o name="app"/>
+  </object>
+phi: '{[[ app -> [[]] ]]}'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)